build/build_mediawiki.sh

```diff
@@ -12,12 +12,15 @@ if [ ! -d "$MEDIAWIKI_DIR" ]; then
    git clone --branch "$MEDIAWIKI_BRANCH_NAME" "$ROOT/git_cache/core.git" "$MEDIAWIKI_DIR"
 fi
 
-# Add vector from cache
-if [ ! -d "$MEDIAWIKI_DIR/skins/Vector" ]; then
-   git clone --branch "$MEDIAWIKI_BRANCH_NAME" "$ROOT/git_cache/skins/Vector.git" "$MEDIAWIKI_DIR/skins/Vector"
+# remove any existing folders
+if [ -d "$MEDIAWIKI_DIR/skins/Vector" ]; then
```

Comment:
No need to check for existence here, `rm -rf` doesn't fail if the directory doesn't exist.

Reply:
correct!